modules/nf-core/sentieon/haplotyper/main.nf

```diff
@@ -32,6 +32,7 @@ process SENTIEON_HAPLOTYPER {
     def args2 = task.ext.args2 ?: ''  // options for the vcf generation
     def args3 = task.ext.args3 ?: ''  // options for the gvcf generation
     def prefix = task.ext.prefix ?: "${meta.id}"
+    def input_list = input.collect{"-i $it"}.join(' ')
```

Comment:
it should work without lists as well as is (at least all the tests pass with and without it being a list). In any case, I liked your solution better because you cleaned up the remaining syntax